### DISCUSSION TITLE
Is the image displayed in the simulator rendered?

### DISCUSSION THREAD

**Discussion by IchbinGYH** [surgical-robotics-ai/surgical_robotics_challenge#59] [Q&A]
Hi, I'm a college student,a novice in computer vision. I want to know is the image displayed in the simulator rendered? Appreciate it if you could answer my question ：）

**Comment by adnanmunawar** [comment]
Hello, I am not sure if I understand the question. We render the scene graph using OpenGL and this is what is displayed in the simulator.

**Reply by IchbinGYH** [reply]
That's what i mean. My English is not good... Scene graph is rendered, I see. Thanks for your reply!
